tests/create_guardduty_org_admin_account/main.tf

```diff
@@ -0,0 +1,38 @@
+# AWS provider account for the AWS organization
+provider "aws" {
+  region  = "us-east-1"
+  profile = "plus3it-management" # Profile must exist in your .aws/config
```

Comment:
```suggestion
  profile = "aws" # Profile must exist in your .aws/config
```

Reply:
I used these profile names to make it clear which account was being used for what.  it was confusing for me to understand this using the more generic names.  I am trying to make this easier for the next person to understand.  I'll make this change if you really want me to, but I feel like we will lose some understandability.

Reply:
Feel free to write it up in the testing section of the readme. But `aws` and `awsalternate` are based on the conventions in the test framework of the terraform aws provider. We've begun trying to stick as close as we can to those conventions, as we also often end up contributing patches upstream when needed. And it's important for us to learn to become comfortable managing, modifying, and troubleshooting aws profiles for every use case.